src/quo2/components/wallet/token_input/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns quo2.components.wallet.token-input.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.buttons.button.view :as button]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [quo2.foundations.common :as common]
+    [quo2.components.wallet.token-input.style :as style]))
+
+(defn calc-value
+  [crypto? currency token value conversion]
+  (if crypto?
+    (str (get common/currency-label currency) (.toFixed (* value conversion) 2))
+    (str (.toFixed (/ value conversion) 2) " " (string/upper-case (clj->js token)))))
+
+(defn- view-internal
+  [{:keys [theme token currency conversion]}]
+  (let [width   (:width (rn/get-window))
+        value   (reagent/atom 0)
+        crypto? (reagent/atom true)]
+    (fn []
+      [rn/view {:style (style/main-container width)}
+       [rn/view {:style style/amount-container}
+        [rn/view
+         {:style {:flex-direction :row
+                  :align-items    :flex-end}}
+         [rn/image
+          {:style  style/token
+           :source (resources/get-token token)}]
+         [rn/text-input
+          {:placeholder            "0"
+           :placeholder-text-color (colors/theme-colors colors/neutral-40 colors/neutral-50)
```

Comment:
@OmarBasem, if I correctly understand, when using `colors/theme-colors` we are relying on the old theming mechanism, where the theme is stored in an atom. 
But since this component already used `with-theme` approach introduced [here](https://github.com/status-im/status-mobile/pull/16191) and got theme as a prop, it would be coherent to use it for color selection.
Please, correct me if I'm wrong.

cc @J-Son89

Reply:
Exactly @vkjr! 😎

Reply:
Eventually the 3rd parameter will be mandatory once it's safe to remove the ratom in there 👍

Reply:
I believe we need one more global version of `colors/theme-colors` with the theme passed as an argument.

Reply:
How do you mean global version?

Reply:
In my branch I have "local" version of `theme-colors` within `style.cljs`, with `theme` argument. And by "global" I mean `colors/theme-colors` that is used everywhere.

Reply:
Oh, I simply misinterpreted the 3rd argument of `colors/theme-colors`) So no additional functions needed in fact)

Reply:
Ah perfect! 👌